backend/src/main/java/hu/flowacademy/munkalap/controller/UserController.java

```diff
@@ -0,0 +1,42 @@
+package hu.flowacademy.munkalap.controller;
+
+import hu.flowacademy.munkalap.dto.UserCreateDTO;
+import hu.flowacademy.munkalap.entity.User;
+import hu.flowacademy.munkalap.exception.WorksheetUserException;
+import hu.flowacademy.munkalap.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import javax.annotation.security.RolesAllowed;
+
+@RestController
+@RequestMapping("api")
+public class UserController {
+
+    private final UserService userService;
+
+    @Autowired
```

Comment:
Ide is mehetne `@RequiredArgsConstructor`

Reply:
Nekem elfér (én jobban szeretem), ha szeretnél taszkot, akkor bátran vegyél fel, ha nem, akkor jelöld pls valamivel, hogy meggondolásra szánod (pl. [think_on_it] v. [suggestion]).

Reply:
Ez egy tipikus boilerplate kódrészlet, én szeretem az ilyeneket eliminálni. De persze, maradhat is, csak legyen konzisztens. 🙂

Reply:
Jó észrevétel, én is hasonlóan gondolom, mint @karolyp, ha már használunk lombokot, használjuk erre is. A service-nél volt hasonló kommentem.